onnxscript/optimizer/_function_folding_test.py

```diff
@@ -114,12 +110,11 @@ def test_single_user_function_is_modified_inplace_after_folding(self):
         """
         )
         optimized = optimizer.optimize(
-            model,
-            onnx_shape_inference=False,
-            num_iterations=1,
+            model, onnx_shape_inference=False, num_iterations=1, inline=False
         )
         self.assertEqual(optimized.functions[0].name, "fun1")
 
+    @unittest.expectedFailure  # FIXME
```

Comment:
This is as designed. This requires creating a copy of the called function to optimize it (without inlining). Our current strategy is to inline and optimize, so this type of optimization is not planned.

Reply:
Removed test case